src/briefcase/platforms/linux/system.py

```diff
@@ -1038,21 +1038,25 @@ def _package_pkg(self, app: AppConfig, **kwargs):
 with details about the release.
 """
             )
-
+            
         # Generate the pkgbuild layout
-        pkgbuild_path = self.bundle_path(app) / "pkgbuild"
+        pkgbuild_path = self.build_path(app) / "pkgbuild"
         with self.input.wait_bar("Generating pkgbuild layout..."):
             if pkgbuild_path.exists():
                 self.tools.shutil.rmtree(pkgbuild_path)
             (pkgbuild_path).mkdir(parents=True)
 
             # Build the source archive
             with self.input.wait_bar("Building source archive..."):
+                
+                # Copy the CHANGELOG file to the bundle_path so it can be included in the source archive
+                self.tools.shutil.copy(changelog_source, self.bundle_path(app) / "CHANGELOG")
+                
                 self.tools.shutil.make_archive(
                     pkgbuild_path / f"{app.app_name}-{app.version}",
                     format="gztar",
-                    root_dir=self.bundle_path(app),
-                    base_dir=f"{app.app_name}-{app.version}",
+                    root_dir=self.build_path(app),
+                    base_dir=self.bundle_path(app),
```

Comment:
This archive will include a pre-build binary unless you either exclude `usr/bin/{app.app_name}` as a path, or disable the call to `make` during the Build command.

Reply:
Hello
I mean, doesn't `briefcase build` just run `make` and `make install`. So, we could just `briefcase create` and then `briefcase package`. If this is not the intended behavior, then I will modify it.

Reply:
I can tell you for free - It's not going to be easy to remove the build step conditionally based on the packaging format, because there's a whole lot of logic that is shared across all backends that determines how to infer that a build is needed. It's going to be a lot easier to skip the specific call to make in the arch packaging case.

Reply:
Sure, skipping the make call should solve the problem.